docs/4.1/components/tooltips.md

```diff
@@ -86,6 +86,19 @@ Trigger the tooltip via JavaScript:
 $('#example').tooltip(options)
 {% endhighlight %}
 
+{% capture callout %}
+##### Usage with overflow `auto` and `scroll`
+
+When the parent container has the CSS property `overflow` set to values `auto` or `scroll` (for example when used within `.table-responsive`) the tooltip has weird flickering behaviour.
```

Comment:
Let's describe what happens instead of saying "it has weird behavior." The current suggestion indicates we have no idea why it happens.

Reply:
> When the parent container has the CSS property `overflow` set to values `auto` or `scroll` (for example when used within `.table-responsive`) the tooltip's placement changes to `bottom` but still keeps original placement's positioning.

but that sounds a bit weird as we're documenting a bug :)

Reply:
We describe a particular use case to help folks